admin/data/sql/migrations/20190620172547.do.restore-vewcheckdiagnostic.sql

```diff
@@ -0,0 +1,37 @@
+IF NOT EXISTS (
```

Comment:
Please turn this back to the standard 'create or alter view ...' statement without an 'EXEC'.